### DISCUSSION TITLE
Play M4a files on SD card

### DISCUSSION THREAD

**Discussion by bkt44** [schreibfaul1/ESP32-audioI2S#98] [Q&A]
Hi
I am using my ESP32-A1S for a webradio, but I would also like to play m4a files on the SD card.
Is it possible to extract the aac from the m4a container and play it?

**Comment by schreibfaul1** [comment]
m4a files cannot be played (syncword not found) , it doesn't seem to be just aac. I believe that the container with the name 'mdat' is the audio data? Anyone who knows the reason or has a solution can answer here.
